Dockerfile

```diff
@@ -19,10 +19,10 @@ RUN apt-get update \
 
 RUN apt -y update
 
-ARG OPENSSL_VERSION=1.1.1l
+ARG OPENSSL_VERSION=1.1.1o
```

Comment:
nit: The latest version of openssl seems to be `1.1.1r`. If security scans for OS packages have not detected any vulnerabilities, then this should be okay to merge.